src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -610,8 +607,7 @@ private void makePlaceholder(final URI uri) throws IOException, FcrepoOperationF
         if (fileForBinaryURI(uri, false).exists() || fileForBinaryURI(uri, true).exists()) {
             response = client().put(uri).body(new ByteArrayInputStream(new byte[]{})).perform();
         } else if (fileForContainerURI(uri).exists()) {
-            response = client().put(uri).body(new ByteArrayInputStream(
-                    "<> a <http://www.w3.org/ns/ldp#Container> .".getBytes()), "text/turtle").perform();
+            response = client().put(uri).body(new ByteArrayInputStream("".getBytes()), "text/turtle").perform();
```

Comment:
It seems like this is basically the same as the preceding block, except for the content type and the way an empty byte array is arrived at. It might make sense to merge the two, or create the body content in the same way in both cases?

Reply:
Sure - I think that makes sense.